ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -61,7 +57,11 @@ export default class StatusDropdown extends React.Component {
     const bugData = bugDetails.data.bugs[0];
 
     const bugVersion = 'Default';
-    const needinfoFrom = bugData.assigned_to;
+    let needinfoFrom = '';
+    if (bugData.assigned_to !== 'nobody@mozilla.org') {
+      needinfoFrom = bugData.assigned_to;
+    }
```

Comment:
If there's no asignee on the ticket then we should set a needinfo for the triage owner.

Reply:
There's also another edge case when the asignee is disabled (here's an [example](https://bugzilla.mozilla.org/show_bug.cgi?id=907925) of an old bug). Just to be safe I think it's best to also check if the user is disabled - if so, then the needinfo should be set to the triage owner.

Reply:
Hi @junngo ! I'll try to clarify your question if I understand correctly what you are asking. The creator(reporter) is a sheriff (including me and other team members), which is different from the person that worked on the patch. By setting a needinfo we are requesting to the person that caused a certain regression to look into it and potentially backout their changes. For more context adding a person to the needinfo field is an esential step in the sheriffing process. If we don't set a needinfo automatically then the sheriff needs to go over the created bug and add it manually which is the oposite of what we want to achieve here.

Reply:
@junngo could you investigate if there's a way to get the details of a certain component in bugzilla? every bug has a component. The component details should contain the triage owner's information (id or something similar).
I've seen that they used this endpoint in the UI https://bugzilla.mozilla.org/rest/component-watching?Bugzilla_api_token=jjt9LqN7XjuFLJUftBra9x so that's were I got the idea.

Reply:
Thanks for reviewing :)
I add the triage owner If there is no assignee.
@beatrice-acasandrei @gmierz